contact-page.php

```diff
@@ -0,0 +1,219 @@
+<?php
+/**
+* Template Name: Contact page
```

Comment:
looks like you have two of these files/templates plus the `send_form_email.php` - is this correct, do you need all 3?